README.md

```diff
@@ -95,8 +95,9 @@ The following snippet shows an example of the JSON response:
 - Select **Firewall and virtual network**, there you can see:  
```

Comment:
At the head of this file, can introduce this content and link back to the RA?

Reply:
Done, please take a look !

Reply:
Looks good, thanks!